### DISCUSSION TITLE
This user account is blocked

### DISCUSSION THREAD

**Discussion by Michaelarinze** [vercel/community#4229] [Account Issue] [answered]
How can I get pass this , my first time registration and then all this?

**Comment by hungchihsueh** [comment]
I'm facing the same error message !
please send help @vercel-infra

**Reply by mum33n** [reply]
Same here, I have been trying to log in my account with github and I keep getting user account is blocked

**Comment by ccdexs** [comment]
Same here, I have been trying to log in my account with github and I keep getting user account is blocked。

**Comment by amyegan** [comment, marked as answer]
Sorry you're having trouble signing up for a new account. We can't safely discuss account information in public because of security concerns. You may have received an email with more details about the error and instructions on next steps. 

You can email [registration@vercel.com](mailto:registration@vercel.com) to open a ticket with our support team. For more info, please check this guide: [Why am I unable to login or signup to the Vercel platform?](https://vercel.com/guides/why-can-i-not-signup)

**Reply by niruallen** [reply]
I contacted them,

I have been waiting for your reply for more than 7 days now!!! Ouch!!!
